src/Arduboy.h

```diff
@@ -63,7 +63,7 @@ class Arduboy : public ArduboyCore
    */
   // void boot(); // defined in core.cpp
 
-  void start() __attribute__ ((deprecated("use begin() instead")));
+  void start() __attribute__((deprecated)) __attribute__ ((warning("use begin() instead")));
```

Comment:
I don't think you need to use `__attribute__` twice.
` void start() __attribute__ ((deprecated, warning("use begin() instead")));`